packages/breadcrumb/src/breadcrumb-item.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <span class="el-breadcrumb__item">
+    <span
+      :class="['el-breadcrumb__inner', to ? 'is-link' : '']"
+      ref="link"
+      role="link"
+    >
+      <slot></slot>
+    </span>
+    <i
+      v-if="separatorClass"
+      class="el-breadcrumb__separator"
+      :class="separatorClass"
+    ></i>
+    <span v-else class="el-breadcrumb__separator" role="presentation">{{
+      separator
+    }}</span>
+  </span>
+</template>
+<script lang="ts">
+import { defineComponent, onMounted, inject, ref, Ref } from 'vue'
+import { propsSymbol } from './propsKey'
+import { useRouter } from 'vue-router'
```

Comment:
If we import `vue-router` in component, we can not add it just into the devdepencency.
We should change the API here.
When user use this component, they should set `globalProperties` to pass the `router` plugin to compoent.
```
   // user component
   VueApp.config.globalProperties.$router = router
   ...

    // our  component
    const instance = getCurrentInstance();
    const router = instance.proxy.$router

   // our logic
    ...
```
Here is an example [runtime-core/src/componentProxy.ts](https://github.com/vuejs/vue-next/blob/master/packages/runtime-core/src/componentProxy.ts#L44)

Reply:
I understand.
the user should add `vue-router-next` dep and add the config below by himself.
So the breadcrumb component can work.

```js
const router = createRouter(/*...*/)

VueApp.use(router)
VueApp.config.globalProperties.$router = router
```